src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let amount = 0;
+  const amountOfDays = days * 40;
+
+  if (days < 3) {
+    amount = amountOfDays;
+  } else if (days >= 3 && days < 7) {
```

Comment:
```suggestion
  } else if (days < 7) {
```
Ты уже проверил что days >= 3 предыдущем if